lib/screens/home/screens/home_screen.dart

```diff
@@ -1,25 +1,369 @@
+                children: [
+                  Text(
+                    'Welcome Back, Jean.!',
+                    style: TextStyle(fontSize: 16, color: kBackgroundColor),
+                  ),
+                  SizedBox(height: 18),
+                  // Text(
+                  //   'Calendar',
+                  //   style: TextStyle(fontSize: 26),
+                  // ),
+                  TableCalendar(
+                    calendarController: _calendarController,
+                    headerVisible: true,
+                    headerStyle: HeaderStyle(
+                        titleTextStyle: TextStyle(fontSize: 26),
+                        titleTextBuilder: (date, locale) =>
+                            DateFormat.MMMM(locale).format(date),
+                        formatButtonVisible: false,
+                        leftChevronVisible: false,
+                        rightChevronVisible: false),
+                    initialCalendarFormat: CalendarFormat.week,
+                    availableGestures: AvailableGestures.horizontalSwipe,
+                    calendarStyle: CalendarStyle(
+                      contentPadding: EdgeInsets.only(top: 20),
+                      weekdayStyle: TextStyle(fontSize: 26),
+                      weekendStyle: TextStyle(fontSize: 26),
+                    ),
+                    daysOfWeekStyle: DaysOfWeekStyle(
+                      weekendStyle: TextStyle(color: kBackgroundColor),
+                      weekdayStyle: TextStyle(color: kBackgroundColor),
+                    ),
+                    builders: CalendarBuilders(
+                      selectedDayBuilder: (context, date, events) => Container(
+                        alignment: Alignment.center,
+                        decoration: BoxDecoration(
+                          color: kButtonColor,
+                          borderRadius: BorderRadius.circular(10),
+                        ),
+                        child: Text(
+                          date.day.toString(),
+                          style:
+                              TextStyle(fontSize: 26, color: kWhiteTextColor),
+                        ),
+                      ),
+                      todayDayBuilder: (context, date, events) => Container(
+                        alignment: Alignment.center,
+                        decoration: BoxDecoration(
+                          color: kBackgroundColor.withOpacity(0.5),
+                          borderRadius: BorderRadius.circular(10),
+                        ),
+                        child: Text(
+                          date.day.toString(),
+                          style: TextStyle(fontSize: 26),
+                        ),
+                      ),
+                    ),
+                  ),
+                  SizedBox(height: 10),
+                ],
+              ),
+            ),
+          ),
+          Expanded(
+            child: SingleChildScrollView(
```

Comment:
Also, please prepare this ListView to render all WoDs from a List<WoD>. You also need to create a data class WoD to represent your models.

I would also like you to use `ListView` instead of `SingleChildScrollView`. This is not wrong at all but `ListView` makes you understand how lists work in a better way.
Follow this link for more info about it. https://flutter.dev/docs/cookbook/lists/long-lists